docs/Data correction documentary-cleared.ipynb

```diff
@@ -0,0 +1,301 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "788a4fce-63dc-4c23-8d2a-e9a05db0fdf7",
+   "metadata": {
+    "jp-MarkdownHeadingCollapsed": true
+   },
+   "source": [
+    "## Data correction procedure for TOF-SANS with polarization analysis\n",
+    "\n",
+    "- Note on notation:\n",
+    "> - subscripts = discretization variable from measurement\n",
+    "> - superscripts = Names\n",
+    "> - () = continuous function of....\n",
+    "\n",
+    "| Value | Meaning | shape | How acquired? | How named in current main branch? |\n",
+    "|:--------:|:--------:|:--------:|:--------:|:--------:|\n",
+    "|  $I^{++}_{t,\\lambda}$   |  uncorrected sample data of ++ channel |  data array with columns for $I$, $\\lambda$, $t$   |  neutron data |Row 1   |\n",
+    "|  $I^{+-}_{t,\\lambda}$   |  uncorrected sample data of +- channel |  data array with columns for $I$, $\\lambda$, $t$   | neutron data    |Row 1   |\n",
+    "|  $I^{-+}_{t,\\lambda}$   |  uncorrected sample data of -+ channel  |  data array with columns for $I$, $\\lambda$, $t$   | neutron data   |Row 1   |\n",
+    "|  $I^{--}_{t,\\lambda}$   |  uncorrected sample data of -- channel |  data array with columns for $I$, $\\lambda$, $t$    | neutron data   |Row 1   |\n",
+    "|  $S^{++}_{t,\\lambda}$   |  corrected sample data of ++ channel  |  data array with columns for $I$, $\\lambda$, $t$   | by pol-correction  |Row 1   |\n",
+    "|  $S^{+-}_{t,\\lambda}$   |  corrected sample data of +- channel  |  data array with columns for $I$, $\\lambda$, $t$   | by pol-correction   |Row 1   |\n",
+    "|  $S^{-+}_{t,\\lambda}$   |  corrected sample data of -+ channel  |  data array with columns for $I$, $\\lambda$, $t$   | by pol-correction   |Row 1   |\n",
+    "|  $S^{--}_{t,\\lambda}$   |  corrected sample data of -- channel  |  data array with columns for $I$, $\\lambda$, $t$   | by pol-correction    |Row 1   |\n",
+    "|  $p^{p|a}$   |  pressure of $^3$He-polarizer/analyzer  |  scalar, 1-dim   | from metadata  |Row 1   |\n",
+    "|  $l^{p|a}$   |  length of $^3$He-polarizer/analyzer   |  scalar, 1-dim   | from metadata   |Row 1   |\n",
+    "|  $T^{g, p|a}(\\lambda)$   |  transmission of empty glass of polarizer/analyzer   |  scalar, 1-dim   | from metadata  |Row 1   |\n",
+    "|  $O^{p|a}(\\lambda)=O^{0, p|a}*\\lambda$   |  wavelength-dependent opacity of $^3$He-polarizer/analyzer  |  continuous function of $\\lambda$   | $O^{0, p|a}$ fit to data |Row 1   |\n",
+    "|  $O^{0, p|a}$   |  wavelength-independent pre-factor of opacity of $^3$He-polarizer/analyzer   |  scalar, 1-dim   | fit to data  |Row 1   |\n",
```

Comment:
The He3 also is not rendered.

Reply:
thanks, its also not in the rest of the text I just see!